packages/safe-ds-eda/src/components/profiling/ProfilingInfo.svelte

```diff
@@ -0,0 +1,116 @@
+            return 'var(--primary-color)';
+        } else if (profilingItem.interpretation === 'error') {
+            return 'var(--error-color)';
+        } else if (profilingItem.interpretation === 'warn') {
+            return 'var(--warn-color)';
+        } else if (profilingItem.interpretation === 'bold') {
+            return 'var(--font-dark)';
+        } else if (profilingItem.interpretation === 'default') {
+            return 'var(--font-light)';
+        } else if (profilingItem.interpretation === 'category') {
+            return 'var(--font-light)';
+        } else {
+            return 'var(--font-light)';
+        }
+    };
+</script>
+
+<div class="wrapper">
+    <div class="profilingItemsTop">
+        {#each profiling.top as profilingTopItem}
+            {#if profilingTopItem.type === 'text'}
+                <div class="profilingItem">
+                    <span style="color: {getProfilingItemColor(profilingTopItem)};">{profilingTopItem.value}</span>
+                </div>
+            {:else if profilingTopItem.type === 'numerical'}
+                <div class="profilingItem">
+                    <span class="profilingItemFirst" style="color: {getProfilingItemColor(profilingTopItem)};"
+                        >{profilingTopItem.name}:</span
+                    >
+                    <span style="color: {getProfilingItemColor(profilingTopItem)};">{profilingTopItem.value}</span>
+                </div>
+            {:else if profilingTopItem.type === 'image'}
+                <div class="profilingItem">
+                    <img
+                        style="width: {imageWidth}px;"
+                        class="profilingImage"
+                        src={'data:image/' + profilingTopItem.value.format + ';base64,' + profilingTopItem.value.bytes}
+                        alt="profiling histogram"
+                    />
+                </div>
+            {/if}
+        {/each}
+    </div>
+    <div class="profilingItemsBottom">
+        {#each profiling.bottom as profilingBottomItem}
+            {#if profilingBottomItem.type === 'text'}
+                <div class="profilingItem">
+                    <span style="color: {getProfilingItemColor(profilingBottomItem)};">{profilingBottomItem.value}</span
+                    >
+                </div>
+            {:else if profilingBottomItem.type === 'numerical'}
+                <div class="profilingItem">
+                    <span class="profilingItemFirst" style="color: {getProfilingItemColor(profilingBottomItem)};"
+                        >{profilingBottomItem.name}:</span
+                    >
+                    <span style="color: {getProfilingItemColor(profilingBottomItem)};">{profilingBottomItem.value}</span
+                    >
+                </div>
+            {:else if profilingBottomItem.type === 'image'}
+                <div class="profilingItem">
+                    <img
+                        style="width: {imageWidth}px;"
```

Comment:
Svelte has a shorthand syntax for styles: `style:width="..."` (https://svelte.dev/docs/element-directives#style-property).